installer/scripts/service_control

```diff
@@ -355,6 +385,15 @@ enable_all_omsagent_services()
        setup_variables ${ws_id}
        enable_omsagent_service
     done
+
+    for ws_id in $NON_OMS_WS
+    do
+        ls -1 $ETC_DIR | grep -w scom > /dev/null 2> /dev/null
```

Comment:
Ditto * 2, I'm thinking this should be a common subroutine, but I'll leave that to you.